src/assets/js/app.js

```diff
@@ -37,8 +37,8 @@ $(document).ready(function(){
 
     $('.js-menu .js-scroll-navigate a').on('click tap', function(){
         $(this).parents('.js-menu').first().removeClass('active');
-        $(this).parents('.js-menu').first().find('a').removeClass('active');
-        $(this).addClass('active');
+        $(this).parents('.js-menu').first().find('a').removeClass('active').removeAttr('aria-current');
+        $(this).addClass('active').attr('aria-current', $(this).text());
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current , the aria-current attribute has to contain a special keyword. In this case, I think `aria-current="true"` is what we are aiming for. all other entries need to have it set to `false`